pyvista/__init__.py

```diff
@@ -19,7 +19,7 @@
 from pyvista.plotting.utilities.sphinx_gallery import _get_sg_image_scraper
 from pyvista.report import GPUInfo, Report, get_gpu_info, vtk_version_info
 from pyvista.themes import (
-    DefaultTheme as _GlobalTheme,
+    DocumentTheme as _GlobalTheme,
```

Comment:
This is the only "breaking" change

Reply:
Should we update `tests/namespace/namespace-pre-refactor*.txt` due to this? The actual type the `_GlobalTheme` name resolves to is different, but we only check for `_GlobalTheme` in the test. This is more of a general question regarding our policy to keep the namespace test inputs up to date.

Reply:
You bring up a good point. I think we should try to leave those files alone as best as possible so that they represent the state of PyVista 0.39 but I have not thought out a good policy around this